### DISCUSSION TITLE
Problems With terminal icons

### DISCUSSION THREAD

**Discussion by felixverdugo** [ohmyzsh/ohmyzsh#10166] [Help] [answered]
Hi community, 
I am having problems with the icons, I have been installed p10k and I don't and p10k is he problem. But I want to ask you in any case.

_Note: I am using WSL_

This is a screenshot using **WINDOWS TERMINAL**
![image](https://user-images.githubusercontent.com/71361387/132396666-84eb8e7f-d873-43e8-9765-834001e0d6f5.png)

This is a screenshot using **Ubuntu Terminal**

![image](https://user-images.githubusercontent.com/71361387/132397790-d197e206-4ece-4ab7-a88b-5fd633e4074c.png)

**Comment by mcornella** [comment, marked as answer]
See https://github.com/romkatv/powerlevel10k#fonts for instructions on how to change fonts.
